charts/values.yaml

```diff
@@ -92,11 +92,26 @@ persistence:
     - ReadWriteOnce
   mountPath: /var/opt/MarkLogic
 
-# specify extra list of volumes
-extraVolumes: []
-# specify extra list of volumeMounts
-extraVolumeMounts: []
-# specify extra list of containerPorts
+## Specify additional list of persistent volume claims
+additionalVolumeClaimTemplates: []
+# - metadata:
+#     name: "logsdir"
+#   spec:
+#     accessModes:
+#       - ReadWriteOnce
+#     resources:
+#       requests:
+#         storage: 10Gi
+
+## specify additional list of volumes
+additionalVolumes: []
```

Comment:
Should this have an example in the comments like the other values?

Reply:
Provided a example:
# - name: additional-vol
#   emptyDir: {}